model/authorization/TestTakerAuthorizationService.php

```diff
@@ -104,6 +113,34 @@ public function isProctored($deliveryId, User $user)
             : $deliverySyncService->isProctoredByDefault();
     }
 
+    /**
+     * @param string $deliveryId
+     * @param string $state
+     * @return bool
+     * @throws common_Exception
+     */
+    public function isActiveUnSecureDelivery($deliveryId, $state)
+    {
+        return $state === DeliveryExecutionInterface::STATE_ACTIVE && !$this->isSecure($deliveryId);
+    }
+
+    /**
+     * @param string $deliveryId
+     * @return bool
+     * @throws common_Exception
+     */
+    private function isSecure($deliveryId)
+    {
+        $delivery = $this->getResource($deliveryId);
+        $activeFeatures = explode(
+            ',',
+            $delivery->getOnePropertyValue(
+                $this->getProperty(DeliveryContainerService::TEST_RUNNER_FEATURES_PROPERTY)
+            )
+        );
+        return in_array('security', $activeFeatures);
```

Comment:
Could you use const instead of string ?  `SecurityFeature::FEATURE_ID`

Reply:
class SecurityFeature marked us deprecated by @hutnikau. I discussed it with him and we decided use string instead constant